### ISSUE TITLE
readers.icebridge not converting 0-360 to -180 to 180

### ISSUE THREAD

**Issue by msmitherdc** [PDAL/PDAL#1214]
```
pdal info --metadata --boundary --stats --filters.hexbin.edge_size=0.2 2013.03.20/ILATM1B_20130320_165003.ATM4BT4.h5

"bbox":
        {
          "maxx": 304.3595581,
          "maxy": 83.65642548,
          "maxz": 30.26199913,
          "minx": 303.2395325,
          "miny": 83.31365967,
          "minz": 20.12599945
        },
```
same for boundary etc.

**Comment by msmitherdc** [comment]
Source data at:  
ftp://n5eil01u.ecs.nsidc.org/SAN2/ICEBRIDGE/ILATM1B.002/2013.03.20/ILATM1B_20130320_165003.ATM4BT4.h5*

**Comment by hobu** [comment]
@msmitherdc please see #1215

**Comment by hobu** [comment]
Fixed in @a1f69b9
